src/bin/main.rs

```diff
@@ -52,6 +52,8 @@ enum SubCommand {
     Shrink {
         #[clap(long, value_parser = ["O0", "O1", "O2", "O3", "O4", "Os", "Oz"])]
         optimize: Option<String>,
+        #[clap(short, long)]
+        keep_name_section: bool,
```

Comment:
Why is this a breaking change. If `keep_name_section` is omitted, it gets false, which is the old behavior?

Reply:
When I use the current `main` and run
```
ic-wasm -o motoko-shrink.wasm motoko.wasm shrink
wasm2wat motoko-shrink.wasm -o motoko-shrink.wat
```
it seems that the name section is kept. 
https://github.com/dfinity/ic-wasm/blob/3246f6d4e888064a3895e2cd88853727933c187d/src/bin/main.rs#L65

If my understanding is correct, the current `shrink` keeps the name sections (and drops the metadata sections). Unless this is a bug.

Reply:
Hmm that's strange, the name section should be dropped here:
https://github.com/dfinity/ic-wasm/blob/3246f6d4e888064a3895e2cd88853727933c187d/src/shrink.rs#L20

Reply:
Name section is special in `walrus` and doesn't appear in the section part. It's controlled by the parser: https://docs.rs/walrus/0.19.0/walrus/struct.ModuleConfig.html#method.generate_name_section